packages/devextreme/js/__internal/ui/collection/m_collection_widget.base.ts

```diff
@@ -876,7 +876,7 @@ const CollectionWidget = Widget.inherit({
     $element.attr(attributes);
   },
 
-  _renderItem(index, itemData, $container, $itemToReplace) {
+  _renderItem(index, itemData, $container, $itemToReplace, ...args) {
```

Comment:
I assume this part is excess or I missed something

Reply:
extended arguments type and used uniqueKey explicitly